File: src/aside/java/org/taktik/icure/dao/FrontEndMigrationDAO.java
package org.taktik.icure.dao;

import org.taktik.icure.entities.FrontEndMigration;

import java.util.List;


public interface FrontEndMigrationDAO extends GenericDAO<FrontEndMigration> {
    List<FrontEndMigration> getByUserIdName(String userId, String name);
}

File: src/main/kotlin/org/taktik/icure/asyncdao/FrontEndMigrationDAO.kt
package org.taktik.icure.asyncdao

import kotlinx.coroutines.flow.Flow
import org.taktik.icure.entities.FrontEndMigration
import java.net.URI

interface FrontEndMigrationDAO: GenericDAO<FrontEndMigration> {
    fun getByUserIdName(userId: String, name: String?): Flow<FrontEndMigration>
}
